packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -132,3 +132,19 @@ def test_if_marathon_app_can_be_debugged(dcos_api_session):
                     assert r['data']['data'] == 'meow', 'Output did not match expected'
                     meowed = True
         assert meowed, 'Read output stream without seeing meow.'
+
+
+def test_files_api(dcos_api_session):
+    app, test_uuid = get_test_app()
+    task_id = 'integration-test-{}'.format(test_uuid)
```

Comment:
This string is hard coded here: https://github.com/dcos/dcos/blob/master/test_util/marathon.py#L12

so, instead of redefining task_id, can we please just use app['id'] in place of task_id? Then we can also not assign test_uuid